templates/account.html

```diff
@@ -19,7 +19,7 @@
               <h1 class="u-no-margin">{{ user.nickname }}</h1>
               <table class="p-table-key-value">
                   <tr><th width="100">Full name</th><td>{{ user.fullname }}</td></tr>
-                  <tr><th>Username</th><td>{{ user.nickname }}</td></tr>
+                  <tr><th>Username</th><td>{{ user.nickname }} <a href="{{ login_url }}" class="p-link--external">Edit</a></td></tr>
```

Comment:
This line starts to be a little bit long:

```html
<tr>
  <th>Username</th>
  <td>
    {{ user.nickname }} <a href="{{ login_url }}" class="p-link--external">Edit</a>
  </td>
</tr>
```

Reply:
And question, do we want to leave s.io ? Because we can't come back from the SSO. Maybe we could open on a new tab?

Reply:
We should avoid opening new tabs etc. where we can, especially if we own the site linked to. New tabs are annoying and anyone that wants to can using normal methods.

Fixed the line length though.